arangod/Aql/AqlCallStack.cpp

```diff
@@ -69,6 +69,16 @@ auto AqlCallStack::popCall() -> AqlCallList {
   return call;
 }
 
+void AqlCallStack::popDepthsLowerThan(size_t depth) {
+  TRI_ASSERT(!_operations.empty());
+  for (auto i = _operations.size() - depth; i < _operations.size(); ++i) {
```

Comment:
Can `depth` be larger here than `_operations.size()`? If yes, then `i` could underflow.
If not, can we at least add an assertion that checks for this?

Reply:
Good point!

It cannot be larger, but this API does not protect us.